lib/assertions/is-string.test.js

```diff
@@ -1,63 +1,54 @@
 "use strict";
 
-var testHelper = require("../test-helper");
+var assert = require("assert");
+var referee = require("../referee");
 
-testHelper.assertionTests("assert", "isString", function(
-    pass,
-    fail,
-    msg,
-    error
-) {
-    pass("for string", "Hey");
-    fail("for object", {});
-    msg(
-        "fail with descriptive message",
-        "[assert.isString] Expected {  } (object) to be string",
-        {}
-    );
-    msg(
-        "fail with custom message",
-        "[assert.isString] Snap: Expected {  } (object) to be string",
-        {},
-        "Snap"
-    );
-    error(
-        "for object",
-        {
-            code: "ERR_ASSERTION",
-            actual: "object",
-            expected: "string",
-            operator: "assert.isString"
-        },
-        {}
-    );
-});
+describe("isString", function() {
+    it("should pass for string", function() {
+        referee.assert.isString("Hey");
+    });
+
+    it("should fail for object", function() {
+        var actual;
+        try {
+            referee.assert.isString({});
+        } catch (err) {
```

Comment:
You could use [assert.throws](https://nodejs.org/api/assert.html#assert_assert_throws_fn_error_message) here.

Reply:
We were trying to not use `referee` to test `referee` :)

Reply:
In referee that would be `assert.exception`. In node assert it's `assert.throws` 😉